tests/acceptance/ConfirmTitleAlignmentSettingsInALCBlockCest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Newsletter;
+
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+
+class ConfirmTitleAlignmentSettingsInALCBlockCest {
+
+  function createWelcomeNewsletter(\AcceptanceTester $I) {
+    $I->wantTo('Confirm title alignment settings in ALC block');
+    // create a post and a newsletter with ALC block
+    $subject = 'ALC Newsletter';
+    $post_title = 'Post for ALC newsletter testing';
+    $post_body = 'Magna primis leo neque litora quisque phasellus nunc himenaeos per, cursus porttitor rhoncus primis cubilia condimentum magna semper curabitur nibh, nunc nulla porttitor aptent aliquet dui nec accumsan quisque pharetra non pellentesque senectus hendrerit bibendum.';
+    $post = $I->cliToArray(sprintf("post create --format=json --porcelain --post_title='%s' --post_content='%s' --post_status='publish' --allow-root", $post_title, $post_body));
+    $I->cli(sprintf("media import https://dummyimage.com/600x400/000/fff.jpg --post_id=%s --title='A downloaded picture' --featured_image --allow-root", $post[0]));
```

Comment:
Isn't it better to add an image for the tests? This creates a dependency on external service.